code/modules/projectiles/projectile/special/temperature.dm

```diff
@@ -32,7 +32,21 @@
 /obj/projectile/temp/cryo
 	name = "cryo beam"
 	range = 9
-	temperature = -240 // Single slow shot reduces temp greatly
+	temperature = -350 // Single slow shot reduces temp greatly
+
+/obj/projectile/temp/cryo/on_hit(atom/target, blocked = 0, pierce_hit)
+	. = ..()
+
+	if(isobj(target))
+		var/obj/objectification = target
+
+		if(objectification.reagents)
+			var/datum/reagents/reagents = objectification.reagents
+			reagents?.expose_temperature(temperature)
```

Comment:
I don't quite see why standard temperature projectiles shouldn't do this too.

Reply:
*shrug* if you'd like this to be on those, that's entirely doable.

Reply:
then dew it.

Reply:
I did that